bee-test/tests/ternary/b1t6.rs

```diff
@@ -1,12 +1,15 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use bee_ternary::{b1t6::encode, T1B1Buf};
+use bee_ternary::{
+    b1t6::{decode, encode},
+    T1B1Buf,
+};
 
 // TODO factorize tests
 
 #[test]
-fn decode() {
+fn b1t6_decode() {
```

Comment:
I don't think it needs the prefix since it's in the b1t6 test module already. And you're not adding the prefix to your encode_decode function. Sorry being picky, it triggers me :)

Reply:
Can't name it `decode` because we import `decode`

Reply:
Not sure why the name is even `decode` because there is only encode used

Reply:
Tbh, I am not a big fan of importing function names, I'd rather import the module and use it. e.g. `b1t6::decode`

Reply:
But yeah, weird that it's named decode, maybe rename it as well ?

Reply:
I can change it, should I name the first test `encode` then?